osgp/shared/osgp-dto/src/main/java/org/opensmartgridplatform/dto/valueobjects/smartmetering/ActualPowerQualityResponseDto.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2021 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.dto.valueobjects.smartmetering;
+
+public class ActualPowerQualityResponseDto extends ActionResponseDto {
+
+    private static final long serialVersionUID = -156966569210717654L;
+
+    private ActualPowerQualityDataDto actualPowerQualityData;
+
+    public ActualPowerQualityDataDto getActualPowerQualityData() {
+        return this.actualPowerQualityData;
+    }
+
+    public void setActualPowerQualityDataDto(ActualPowerQualityDataDto actualPowerQualityData) {
```

Comment:
Missing final modifier with the parameter, required by the coding conventions.

Reply:
Added final modifier